### ISSUE TITLE
[1.21][NeoForge][0.1.5 rc.1] Likely Incompatability with Occultism

### ISSUE THREAD

**Issue by V2LenKagamine** [FiguraMC/Figura#241]
### Bug Description

Crashes in blank test instance with Occultism + dependencies and Figura 0.1.5 RC-1

### Reproduction Steps

Install Blank instance of Neoforge 21.0.109
Install Occultism + dependencies
Install Figura RC-1
Launch. Crash,

### Log File

Log files can be located [On the Occultism Github](https://github.com/klikli-dev/occultism/issues/1159)

**Comment by UnlikePaladin** [comment]
Figura is not referenced anywhere in the log's crash stacktrace, it quite literally says Occultism encountered an error. And please attach the log files as well here next time you open a bug.

**Comment by V2LenKagamine** [comment]
Yep, sorry, was not feeling to great when I posted and figured better to post it in in both parties incase I post it in the wrong one only.

**Comment by UnlikePaladin** [comment]
It's alright, just please do attach the log next time lol

**Comment by klikli-dev** [comment]
@UnlikePaladin is Figura using any mixins? The error happens during the RegisterRenderers event, and is thrown by mixin. 
The crash only appears if figura is installed alongside occultism, not if occultism is alone in packs. Occultism does not use any mixins and just uses the neo register renderer events "as-is". 

Outside of mixins I don't see any way that figura<->occultism interactions could cause this (unless it is a neo error)

**Comment by UnlikePaladin** [comment]
Figura does use Mixins in a lot of places but does not touch Neo internals like RegisterRenderers at all, if the issue is only happening when both are installed then it might be encountering an error earlier and crashes when it reaches the event, so a full game log and not just the crash log would be needed

**Comment by klikli-dev** [comment]
@V2LenKagamine can you provide the full log?

**Comment by V2LenKagamine** [comment]
[crash-2024-07-19_13.09.24-client.txt](https://github.com/user-attachments/files/16352442/crash-2024-07-19_13.09.24-client.txt)
[debug.log](https://github.com/user-attachments/files/16352444/debug.log)
[latest.log](https://github.com/user-attachments/files/16352445/latest.log)
These should be the relevant logs for the error, This occured in an instance of JUST occultism[1.136.5](And dependents.) + Fig.[0.1.5 RC 1]

**Comment by mcmeiler** [comment]
Issue still persists on NeoForge 1.21.1 with Figura 0.1.5 RC3 and Occultism 1.161.3.

[latest.log](https://github.com/user-attachments/files/17311047/latest.log)
[debug.log](https://github.com/user-attachments/files/17311049/debug.log)
[crash-2024-10-09_17.38.27-client.txt](https://github.com/user-attachments/files/17311052/crash-2024-10-09_17.38.27-client.txt)
Should be all the relevant logs. Let me know if there is anything else you need to know.
